database/enums/enum_workspace_homedir_location.pg

```diff
@@ -0,0 +1 @@
+s3, efs
```

Comment:
Rather than defining a new enum, perhaps we could use the existing `enum_file_storage_type`? It was created to solve a similar problem (but in the reverse FileSystem -> S3) direction. In any case I'd prefer to call the non-S3 version something that isn't specific to EFS (so maybe 'filesystem' or 'disk' or similar), because from PL's point of view these are just local files (which will happen to be on a mounted EFS share).

Reply:
Ah good idea, should've known there was an existing enum for it.